### PR TITLE
updated createDevon4ngProject function

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
 ***
 
 ### createDevon4ngProject <a name="createDevon4ngProject"></a>
+This function is used to create a devon4ng project ib workspace.
```
Comment (GuentherJulian) [inline documentation/Functions.md:405]: You can additionally link `devon4ng` with the Github repository https://github.com/devonfw/devon4ng

File: documentation/Functions.md [hunk 2]
```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
...
 #### parameter 
-1. Name of the Project.
-2. Path to where the Project should be created (relative to workspace). Folder should exist.
-3. (Optional) String array of parameters, according to https://angular.io/cli/new.
+You need three parameters for this function.
+Example: createDevon4ngProject(1st-parameter, 2nd-parameter, 3rd-parameter)
+
+1st-parameter: It is the name of the Project.It is of type String.
```
Comment (GuentherJulian) [inline documentation/Functions.md:410]: project

File: documentation/Functions.md [hunk 3]
```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
...
+
+2nd-parameter: It is the path to where the Project should be created (relative to workspace). Folder should exist. It is of type String.
```
Comment (GuentherJulian) [inline documentation/Functions.md:412]: project

File: documentation/Functions.md [hunk 4]
```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
...
+
+3rd-parameter: It is an optional parameter. It contains String array of parameters, according to https://angular.io/cli/new. You can pass argument that can be used in angular CLI.
```
Comment (GuentherJulian) [inline documentation/Functions.md:414]: Angular CLI

File: documentation/Functions.md [hunk 5]
```diff
@@ -402,19 +402,27 @@ adaptTemplatesCobiGen()
...
-createDevon4ngProject("exampleAngularProject", "projects", ["--verbose"])
+2. createDevon4ngProject("exampleAngularProject", "projects", ["--verbose"])
 Will create the angular project to the directory projects within the current workspace and adds more details to output logging.
 
-#### Details
-This command also works if the devonfw IDE is not installed, but then you have to make sure that the Angular cli is installed.
+Note:
+1. No background script will run. Katacoda user will have to manually execute the command.
+
+2. This command also works if the devonfw IDE is not installed, but then you have to make sure that the Angular cli is installed.
```
Comment (GuentherJulian) [inline documentation/Functions.md:425]: devonfw-ide
Angular CLI

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 2 above)
Comment (dixyushi) [reply documentation/Functions.md:410]: done

File: documentation/Functions.md (hunk 1 above)
Comment (dixyushi) [reply documentation/Functions.md:405]: done

File: documentation/Functions.md (hunk 3 above)
Comment (dixyushi) [reply documentation/Functions.md:412]: changed

File: documentation/Functions.md (hunk 4 above)
Comment (dixyushi) [reply documentation/Functions.md:414]: changed.

File: documentation/Functions.md (hunk 5 above)
Comment (dixyushi) [reply documentation/Functions.md:425]: Changed.

File: documentation/Functions.md [hunk 6]
```diff
@@ -402,19 +402,39 @@ adaptTemplatesCobiGen()
...
-Will create the angular project to the directory projects within the current workspace and adds more details to output logging.
+2. createDevon4ngProject("exampleAngularProject", "projects", ["--verbose"])
+Will create the Angular project to the directory projects within the current workspace and adds more details to output logging.
 
-#### Details
-This command also works if the devonfw IDE is not installed, but then you have to make sure that the Angular cli is installed.
+Note:
+1. No background script will run. 
+
+2.The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:433]: Add space after `2.`
Katacoda

File: documentation/Functions.md [hunk 7]
```diff
@@ -402,19 +402,39 @@ adaptTemplatesCobiGen()
...
+
+3. This requires devonfw-ide to be installed for their execution.
+
+4. This command also works if the devonfw-ide is not installed, but then you have to make sure that the Angular CLI is installed.
```
Comment (GuentherJulian) [inline documentation/Functions.md:437]: Point 3 and 4 contradict each other. I would just remove the third point

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 6 above)
Comment (dixyushi) [reply documentation/Functions.md:433]: changed!!

**Pull request merged**